besu/src/main/java/org/hyperledger/besu/cli/options/MiningOptions.java

```diff
@@ -52,27 +52,34 @@
 /** The Mining CLI options. */
 public class MiningOptions implements CLIOptions<MiningConfiguration> {
 
+  private static final String DEPRECATION_PREFIX =
+      "Deprecated. PoW consensus is deprecated. See CHANGELOG for alternative options. ";
+
   @Option(
       names = {"--miner-enabled"},
-      description = "Set if node will perform mining (default: ${DEFAULT-VALUE})")
+      description =
+          DEPRECATION_PREFIX + "Set if node will perform mining (default: ${DEFAULT-VALUE})")
```

Comment:
this is used for QBFT so may need a rename rather than deprecation

Reply:
I was wrong, it isn't used for QBFT actually

Reply:
Yep QBFT doesn't use the start/stop CLI options they aren't ignored so it's fine to deprecate this.

Reply:
awesome, thanks both for confirming